### DISCUSSION TITLE
shortlinks are just single digit numbers

### DISCUSSION THREAD

**Discussion by adnaanqureshi** [YOURLS/YOURLS#2954] [Q&A] [answered]
hi
fresh install into a subdirectory on the root of my server
admin works fine etc
can generate shortlinks but they are all just coming out as /1  and then /2 and then /3 etc 
when these are clicked, the server seems to ignore the /x and just shows me a directory listing of the folder
ive changed it from 32 to 64 etc but no difference.
not sure what im doing wrong?

**Comment by dgw** [comment, marked as answer]
This is two different questions rolled into one.

If you want random link keywords instead of the sequential ones you get now, activate the Random Keywords plugin.

For why your install shows a directory listing instead of following the short links, you'll first have to give us a lot more detail about your setup—what server, PHP version, ideally the server configuration for this domain.

**Reply by adnaanqureshi** [reply]
thanks! yeh ill add the plugin. hopefully that will solve both issues!

**Reply by ozh** [reply]
Nope. Two unrelated issues as @dgw said

**Reply by adnaanqureshi** [reply]
adding the plugin has made it work perfectly now! thanks!

**Reply by dgw** [reply]
I'm confused as hell but I'll take the win. 🤷‍♂️

**Reply by adnaanqureshi** [reply]
haha yeh i must have some strange server setting. ps - does the api work with file get contents and a GET query string (url encoded of course) ?
